package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@acpr/rate-limit-postgresql",
-  "version": "1.1.1",
+  "version": "2.0.0",
```

Comment:
When I do version changes, I like to use the `npm version major|minor|patch` command rather than changing package.json manually, because it does some extra steps, including creating a git tag. (You can then push the tag to github with `git push --follow-tags`)

Reply:
Will take this into account in the future, thanks.